src/odemis/driver/smartpod.py

```diff
@@ -644,6 +671,8 @@ def moveRel(self, shift):
         if not shift:
             return model.InstantaneousFuture()
 
```

Comment:
For `moveRel()` and `moveAbs()`, you also want to call `shift = self._applyInversion(shift)`, in order to support the "inverted" argument. It's also needed on the `updatePosition()`, just before setting `.position`. This is a very handy argument in case the actuator vendor had a different convention than us on the "direction" of each axis (which is quite likely, as it's mostly arbitrary)